dango/testing/tests/lending_pool.rs

```diff
@@ -0,0 +1,473 @@
+        )
+        .should_fail_with_error("Only the owner can delist denoms");
+
+    // Delist a denom from owner, should succeed
+    suite
+        .execute(
+            &mut accounts.owner,
+            contracts.lending_pool,
+            &lending_pool::ExecuteMsg::DelistDenom(USDC.clone()),
+            Coins::new(),
+        )
+        .should_succeed();
+}
+
+#[test]
+fn whitelist_denom_works() {
+    let (mut suite, mut accounts, _codes, contracts) = setup_test();
+
+    // Ensure USDC is already in the whitelist
+    suite
+        .query_wasm_smart(contracts.lending_pool, QueryWhitelistedDenomsRequest {
+            limit: None,
+            start_after: None,
+        })
+        .should_succeed_and_equal(vec![USDC.clone()]);
+
+    // Try to whitelist a denom that is already in the whitelist, should fail
+    suite
+        .execute(
+            &mut accounts.owner,
+            contracts.lending_pool,
+            &lending_pool::ExecuteMsg::WhitelistDenom(USDC.clone()),
+            Coins::new(),
+        )
+        .should_fail_with_error("Denom already whitelisted");
+
+    // Whitelist ATOM from owner, should succeed
+    suite
+        .execute(
+            &mut accounts.owner,
+            contracts.lending_pool,
+            &lending_pool::ExecuteMsg::WhitelistDenom(ATOM.clone()),
+            Coins::new(),
+        )
+        .should_succeed();
+
+    // Ensure ATOM is now in the whitelist
+    suite
+        .query_wasm_smart(contracts.lending_pool, QueryWhitelistedDenomsRequest {
+            limit: None,
+            start_after: None,
+        })
+        .should_succeed_and_equal(vec![USDC.clone(), ATOM.clone()]);
+}
+
+#[test]
+fn delist_denom_works() {
+    let (mut suite, mut accounts, _codes, contracts) = setup_test();
+
+    // Ensure USDC is already in the whitelist
+    suite
+        .query_wasm_smart(contracts.lending_pool, QueryWhitelistedDenomsRequest {
+            limit: None,
+            start_after: None,
+        })
+        .should_succeed_and_equal(vec![USDC.clone()]);
+
+    // Delist denom not in the whitelist, should fail
+    suite
+        .execute(
+            &mut accounts.owner,
+            contracts.lending_pool,
+            &lending_pool::ExecuteMsg::DelistDenom(ATOM.clone()),
+            Coins::new(),
+        )
+        .should_fail_with_error("Denom not whitelisted");
+
+    // Delist USDC from owner, should succeed
+    suite
+        .execute(
+            &mut accounts.owner,
+            contracts.lending_pool,
+            &lending_pool::ExecuteMsg::DelistDenom(USDC.clone()),
+            Coins::new(),
+        )
+        .should_succeed();
+
+    // Ensure USDC is no longer in the whitelist
+    suite
+        .query_wasm_smart(contracts.lending_pool, QueryWhitelistedDenomsRequest {
+            limit: None,
+            start_after: None,
+        })
+        .should_succeed_and_equal(vec![]);
+}
+
+#[test]
+fn cant_deposit_from_margin_account() -> anyhow::Result<()> {
```

Comment:
We like to `unwrap` in tests instead of errors. Let's keep this practice consistent

Reply:
Alright. Any practical reason or just style? Personally find it ugly to have unwraps everywhere.

Reply:
Just style